dao/src/main/java/com/fit_goal/domain/EventDto.java

```diff
@@ -0,0 +1,33 @@
+package com.fit_goal.domain;
+
+
+import lombok.*;
+import org.bson.types.ObjectId;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@Data
+@NoArgsConstructor
+public class EventDto {
```

Comment:
I propose to use Audit instead of Event, what we want is to track what was the actions that user did, and the best fit  for that might be AuditDto, AuditDao, AuditService and audit collection in mongodb

Reply:
refactored